Gemfile.lock

```diff
@@ -481,6 +481,3 @@ DEPENDENCIES
   uglifier
   unf
   unicorn
-
-BUNDLED WITH
```

Comment:
You shouldn't remove that. Update your version of bundler instead: `gem install bundler`